src/mlpack/methods/ann/convolution_rules/svd_convolution.hpp

```diff
@@ -87,13 +87,13 @@ class SVDConvolution
         NaiveConvolution<BorderMode>::Convolution(subOutput, U.unsafe_col(0),
             output);
 
+        arma::Mat<eT> temp;
```

Comment:
I don't disagree with this change, but just as a note, the time it takes to construct the object probably pales in comparison to the time `Convolution()` takes, so this isn't likely to make any noticeable difference.  Still, it is not a bad idea to do. :)